qa-include/pages/admin/admin-categories.php

```diff
@@ -218,16 +218,35 @@
 					$recalc = $hassubcategory && $inslug !== $editcategory['tags'];
 				}
 
+				qa_report_event('cat_edit', qa_get_logged_in_userid(), qa_get_logged_in_handle(), qa_cookie_get(), array(
+					'categoryid' => $editcategory['categoryid'],
+					'parentid' => @$inparentid,
+					'content' => @$incontent,
+					'position' => @$inposition,
+					'name' => $inname,
+					'slug' => $inslug
+				));
+
 				qa_redirect(qa_request(), array('edit' => $editcategory['categoryid'], 'saved' => true, 'recalc' => (int)$recalc));
 
 			} else { // creating a new one
 				$categoryid = qa_db_category_create($inparentid, $inname, $inslug);
+				
```

Comment:
If the HTML in GitHub is not playing tricks on me, I see tabs in this line. It should be adding whitespace errors. I'd rather remove the line (2 empty lines as separators are IMO, a bit overkill).